stages/io.weldr.systemd

```diff
@@ -1,16 +1,23 @@
 #!/usr/bin/python3
 
 import json
-import os
 import subprocess
 import sys
 
+
 def main(tree, options):
     enabled_services = options["enabled_services"]
+    disabled_services = options["disabled_services"]
 
     for service in enabled_services:
         subprocess.run([f"systemctl", "--root", tree, "enable", service], check=True)
 
+    for service in disabled_services:
+        subprocess.run([f"systemctl", "--root", tree, "disable", service], check=True)
+
+    return 0
```

Comment:
* Why do we need to disable services? I thought nothing's enabled by default coming from packages?
* use `options.get("disabled_services", [])` — both options are optional (haha!)
* Please put this into a separate commit.

Reply:
done